src/style.css

```
@@ -1 +1,69 @@
 /* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  --color-blue: #00acdc;
+  cursor: pointer;
+}
```

```suggestion
body {
  margin: 0;
  --color-blue: #00acdc;
}
```
body doesn't have default paddings. no need to make cursop pointer for all body tag)